src/cairoWriter/writers/assignmentWriter.ts

```diff
@@ -19,7 +19,7 @@ export class AssignmentWriter extends CairoASTNodeWriter {
       ) &&
       !(lhs instanceof TupleExpression)
     ) {
-      return [`let (${nodes[0]}) ${node.operator} ${nodes[1]};`];
+      return [`let ${nodes[0]} ${node.operator} ${nodes[1]};`];
     }
```

Comment:
It seems regardless of the condition the result is the same now, so let's remove `if`

Reply:
Also is nodes[0] same as `rhs`? Then we should use [`let ${nodes.rhs} ${node.operator} ${node.lhs}`

Reply:
nodes[0] and nodes[1] are the string representation for those expressions given by the writer

Reply:
Oh I see, then it's good. Please take a look at the return inside and outside(below) the `if`. They seem the same, thus I guess the if is redundant